web/META-INF/resources/index.html

```diff
@@ -12,95 +12,40 @@
-            </span>
-                    <button id="searchButton" class="btn btn-search"
-                            type="button">
-                        <i class="fa fa-search"></i>&nbsp;&nbsp;<span
-                            class="bold">Search</span>
-                    </button>
-                </p>
-                <div id="searchOption" style="padding: 10px 0px 0px 20px;">
-                    <span>Search Operator: </span>
-                    <div class="radio radio-inline">
-                        <input type="radio" id="searchOption_And" value="and"
-                               name="searchOption" checked="checked"> <label
-                            for="searchOption_And"> And </label>
-                    </div>
-                    <div class="radio radio-inline">
-                        <input type="radio" value="or" id="searchOption_Or"
-                               name="searchOption"> <label
-                            for="searchOption_Or"> Or </label>
-                    </div>
-                    <div class="radio radio-info radio-inline">
-                        <input type="radio" value="phrase"
-                               id="searchOption_Phrase" name="searchOption"> <label
-                            for="searchOption_Phrase"> Phrase </label>
-                    </div>
-                </div>
-            </form>
-        </div>
-    </div>
-</div>
-<footer>
-    <div class="footer-content">
-        <p>
-            <a href="https://github.com/mudrod/mudrod"
-               target="_blank">MUDROD</a>
-            is funded by NASA AIST Program(NNX15AM85G), developed by <a
-                href="http://stcenter.net/stc/">NSF Spatiotemporal
-            Innovation Center</a> in collaboration with <a
-                href="http://www.jpl.nasa.gov/">NASA JPL</a>.
-        </p>
-    </div>
-    <script src="js/jquery-2.1.1.js"></script>
-    <script src="js/bootstrap/3.3.6/bootstrap.min.js"></script>
-    <script type="text/javascript" src="js/typeahead.bundle.min.js"></script>
-    <script src="js/index.js"></script>
-</footer>
-</body>
-</html>
\ No newline at end of file
+<html lang="en" ng-app="mudrodApp"> 
+    <head> 
+        <title>MUDROD</title>             
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> 
+        <link rel="icon" href="images/logo_nasa.png">
+        <!-- link rel="stylesheet" href="lib-css/bootstrap.min.css"  media="screen"/--> 
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+        <link rel="stylesheet" href="css/styles.css"  media="screen"/> 
+        <link href="css/index.css" rel="stylesheet">
+        
+        <script src="js/libs/jquery-1.10.2.min.js"></script>
+        <!-- script src="js/libs/bootstrap.min.js"></script -->  
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)